src/App/App.tsx

```diff
@@ -231,34 +250,44 @@ const App = () => {
                 <ConnectedPrivateRoute
                   redirectPath={APP_CALLBACK_URL}
                   disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                  routerEnabled={ENABLE_MDA_POINT}
+                  routerDisabledRedirectPath={PAGE_NOT_FOUND_URL}
```

Comment:
Should we create an issue to look into making this more DRY?  Maybe by using a provider or some other thing?

Reply:
#1618